io.embrace.sdk/Editor/Editor Windows/Installation.cs

```diff
@@ -114,10 +115,8 @@ public static void InitializeOnLoad()
                 }
             }
 
-            if (!EmbraceProjectSettings.User.GetValue<bool>(nameof(DeviceSDKInfo.isManifestSetup)))
-            {
-                SetupManifest(sdkInfo);
-            }
+            SetupManifest(sdkInfo); // TODO: For later, this is inefficient as we should only do this once. But it's fine as this deals specifically with upgrading.
```

Comment:
just want to confirm since the comment it saying this is dealing with upgrading, but since it's outside of the `if (sdkInfo.version != lastUsedVersion)` this would happen in other cases as well correct?

Reply:
This would. This is the best quick fix I could think of for users who were upgrading, as the installation is designed to only run once. and in this case we actually need to guarantee it has run in order to automatically fix the user's local setup.